protocol/x/bridge/types/codec.go

```diff
@@ -3,35 +3,12 @@ package types
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-func RegisterCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(&MsgAcknowledgeBridges{}, "bridge/AcknowledgeBridges", nil)
```

Comment:
the RegisterConcrete calls that we have are incomplete. Some modules have them, some don't. The ones that do only call RegisterConcrete for some of the Msgs because they were not updated when new Msgs were added. Don't think it is really important to keep these tbh since they are in a weird incomplete state

Reply:
Okay, can you share which modules + msgs are missing the `RegisterConcrete`? 

Trying to better understand if they are not registered intentionally.

Reply:
modules missing RegisterConcrete for message on `main`

clob: missing all of the new messages we have added for governance
vest: missing `MsgDeleteVestEntry`
prices: missing `MsgCreateOracleMarket`
blocktime: missing `MsgUpdateDowntimeParams` and `MsgIsDelayedBlock`
perpetuals: missing `MsgCreatePerpetual`

Reply:
I see. I don't see an intentional pattern. For example, `vest` should register both `MsgDeleteVestEntry` + `MsgSetVestEntry`.

I agree that we can delete these for now and add them back later